charts/incubator/zammad/questions.yaml

```diff
@@ -0,0 +1,200 @@
+              - variable: webcon_schedprocesses
+                label: Independent Process for Scheduled Jobs
+                description: Allows you to enable or disable independent processes for jobs such as LDAP syncs. This can free up the Zammad background worker for other tasks when some scheduled tasks are fairly long.
+                schema:
+                  type: boolean
+                  default: false
+              - variable: webcon_delayedjobs
+                label: Processes for Delayed Jobs
+                description: Allows you to specify the amount of processes that should work on delayed jobs. Increasing this can improve issues with delayed jobs stacking up in your system. You may want to try to use `Concurrent Session Jobs` before though. This option can be very CPU intensive. The maximum value is 16.
+                schema:
+                  type: int
+                  default: 0
+        - variable: rails
+          label: Rails Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: rails_trustedproxies
+                label: Rails Trusted Proxies
+                description: Allows you to change the trusted proxies. Changing this isn't recommended unless you know what you're doing.
+                schema:
+                  type: list
+                  default: ['127.0.0.1', '::1']
+                  items:
+                    - variable: url
+                      label: URL
+                      schema:
+                        type: string
+                        required: true
+                        default: ""
+        - variable: redis
+          label: Redis Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: redis_url
+                label: Redis URL
+                description: Allows you to store your web socket conection within Redis, and should point to your Redis instance. The default value assumes your Redis chart's name is redis. If not provided, Zammad falls back to the file system (`/opt/zammad/tmp/websocket_*`).
+                schema:
+                  type: string
+                  default: "http://redis.ix-redis.svc.cluster.local:6379"
+        - variable: memcached
+          label: Memcached Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: memcached_url
+                label: Memcached URL
+                description: Allows you to store your application cache files within Memcached, and should point to your Memcached instance. The default value assumes your Memchaced chart's name is memcached. If not provided, Zammad falls back to the file system (`/opt/zammad/tmp/cache*`).
+                schema:
+                  type: string
+                  default: "http://memcached.ix-memcached.svc.cluster.local:11211"
```

Comment:
whats the deal with this?

Reply:
Yep not necessary in the questions at all TBH

Reply:
Fixing and will commit shortly. How should elasticsearch be handle in regards to the params such as the SHEMA/PORT?
![image](https://github.com/truecharts/charts/assets/9328458/6549e2f2-986c-4697-9f33-08e359bcd463)

Reply:
Marking as resolved, answered in stavros' review below.